client/components/common/ContentPreview/Element.vue

```diff
@@ -1,19 +1,40 @@
 <template>
-  <div @click="toggleSelection" class="element-preview-container">
+  <div :class="elementWidth" class="element-preview-container">
```

Comment:
Please consider if this approach would make it more straightforward to infer what's going on with the layout straight from reading the template:
```suggestion
  <div :class="col-xs-`${elementWidth} float-none`" class="element-preview-container">
```